server/publicapi/items/service.py

```diff
@@ -95,6 +95,48 @@ def get(self, req, lookup):
 
         return super().get(req, lookup)
 
+    def on_fetched_item(self, document):
+        """Event handler when a single item is retrieved from database.
+
+        It sets the item's `uri` field and removes all the fields added by the
+        `Eve` framework that are not part of the NINJS standard (except for
+        the HATEOAS `_links` object).
+
+        :param dict document: fetched MongoDB document representing the item
+        """
+        self._set_uri(document)
+
+        for field_name in ('_id', '_etag', '_created', '_updated'):
```

Comment:
If `_etag` is not in the payload how do you know if a new version is up? Do we use a different field to specify the version?